loader/checkpoint.go

```diff
@@ -416,17 +435,10 @@ func (cp *RemoteCheckPoint) Count(tctx *tcontext.Context) (int, error) {
 }
 
 func (cp *RemoteCheckPoint) String() string {
-	// `String` is often used to log something, it's not a big problem even fail,
-	// so 1min should be enough.
-	tctx2, cancel := cp.logCtx.WithTimeout(time.Minute)
-	defer cancel()
-
-	if err := cp.Load(tctx2); err != nil {
```

Comment:
Do not need to `Load` even for the first time, right?

Reply:
I guess checkpoint are not cached in memory before, so this `String()` need to fetch from downstream DB to show latest checkpoint. now it's no need